### DISCUSSION TITLE
蓝灯能不能不要自动切换到US节点

### DISCUSSION THREAD

**Discussion by keene06** [getlantern/lantern-forum-cn#541] [General]
刚打开vpn的时候，默认是使用US节点，是完全连不上网的，等好几分钟才会切到SG节点，可以上网了，但是每过一会就会自动切换到US，又要重复第一步的流程

**Comment by MaxMKMax** [comment]
@keene06 快要可以自行选择服务器了. 技术团队在搞着了. 

很抱歉, 现在暂且不可.

**Reply by szhero** [reply]
求求了快点安排上吧，这节点一分钟能自动切十次，各种跳真人验证

**Reply by MaxMKMax** [reply]
@szhero 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
